apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,80 @@
+
+suite('Player View Test', function () {
+  var pv, ratings;
+  var realL10n = navigator.mozL10n;
+  function testRatingsAriaChecked(checkedIndex) {
+    pv.setRatings(checkedIndex);
+    Array.prototype.forEach.call(ratings, function (rating, index) {
+      assert.equal(checkedIndex - 1 === index ? 'true' : 'false',
+        rating.getAttribute("aria-checked"));
+    });
+  }
+
+  suiteSetup(function () {
+    navigator.mozL10n = MockL10n;
+    var starMarkUp = '<div id="views-player" class="view hidden">' +
+      '<div id="player-cover">' +
+      '<div id="player-cover-image"></div>' +
+      '<div id="player-cover-caption">' +
+      '<div id="player-cover-artist" data-l10n-id="songArtist">Song Artist</div>' +
+      '<div id="player-cover-album" data-l10n-id="songAlbum">Song Album</div>' +
+      '<button id="player-cover-share" class="player-cover-button" data-icon="share" data-l10n-id="share-song"></button>' +
+      '</div>' +
+      '<div id="player-cover-buttons">' +
+      '<button id="player-album-repeat" class="player-cover-button" data-icon="repeat"></button>' +
+      '<div id="player-album-rating" data-l10n-id="player-album-rating" role="radiogroup">' +
+      '<button class="rating-star" data-rating="1" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"1"}\'></button>' +
+      '<button class="rating-star" data-rating="2" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"2"}\'></button>' +
+      '<button class="rating-star" data-rating="3" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"3"}\'></button>' +
+      '<button class="rating-star" data-rating="4" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"4"}\'></button>' +
+      '<button class="rating-star" data-rating="5" role="radio" data-l10n-id="rating-star" data-l10n-args=\'{"n":"5"}\'></button>' +
+      '</div>' +
+      '<button id="player-album-shuffle" class="player-cover-button" data-icon="shuffle"></button>' +
+      '</div>' +
+      '<section id="info-banner" role="status">' +
+      '<p data-l10n-id="music-paused-notification"></p>' +
+      '</section>' +
+      '</div>' +
+      '<div id="player-seek">' +
+      '<span id="player-seek-elapsed" dir="ltr">00:00</span>' +
+      '<div id="player-seek-bar">' +
+      '<progress id="player-seek-bar-progress" value="0"></progress>' +
+      '<div id="player-seek-bar-indicator"></div>' +
+      '</div>' +
+      '<span id="player-seek-remaining" dir="ltr">00:00</span>' +
+      '</div>' +
+      '<div id="player-controls" dir="ltr">' +
+      '<button id="player-controls-previous" class="player-controls-button" data-icon="skip-back"></button>' +
+      '<button id="player-controls-play" class="player-controls-button is-pause" data-icon="pause"></button>' +
+      '<button id="player-controls-next" class="player-controls-button" data-icon="skip-forward"></button>' +
+      '</div>' +
+      '<audio id="player-audio" class="hidden"></audio>' +
+      '</div>';
```

Comment:
I don't know the unit test well, but is it possible to actually load the index.html instead of marking up them? I feel here is kind of duplicating the html and probably we can just maintain index.html, if we need to modify these tests.